db/migrate/20231123035111_create_books.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateBooks < ActiveRecord::Migration[7.1]
+  def change
+    create_table :books do |t|
+      t.string :name
```

Comment:
We already add `validate :name, presence: true` to ensure that all the books have their name. That is an application level validation. We can also add a database level validation as well.

For example,

```ruby
t.string :name, null: false
```

ref: https://guides.rubyonrails.org/active_record_migrations.html#creating-a-table